torchvision/prototype/datapoints/_image.py

```diff
@@ -169,6 +170,12 @@ def elastic(
         )
         return Image.wrap_like(self, output)
 
+    def to_grayscale(self, num_output_channels: Literal[1, 3] = 1) -> Image:
```

Comment:
Should the method be called `rgb_to_grayscale()` instead? Or just `grayscale()`?

Right now the class is called `Grayscale()`, the functionals are called `rgb_to_grayscale...()`. Keeping `to_grayscale()` for the methods seem like adding even more inconsistency, but IDK which one to go for.

We also have the `to_grayscale()` functional outlier in the v1 area which only works on PIL images (that we probably want to deprecate as discussed before).

Reply:
shoot, I merged before we got a chance to address that. Any thought @pmeier?

Reply:
I would keep consistency with the dispatchers, i.e. naming this `rgb_to_grayscale`.

Reply:
Following up in https://github.com/pytorch/vision/pull/7136